ZipBuilder/Template/FrameworkMaker.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,240 @@
+// !$*UTF8*$!
```

Comment:
Probably should eventually migrate to the CocoaPods packager instead of this

Reply:
+1, SGTM.